### PR TITLE
Fix codecov inconsistencies

### LINKED ISSUE #810: inconsistencies with CodeCov rejections
Possibly in relation to #747, I'm seeing that some PRs are blocked if they have no tests, which is fine, but that when adding tests, 

1. we can't seem to get the Codecov reported coverage for the PR to change
2. CodeCov's info seems inconsistent; initially this PR (https://github.com/publiclab/mapknitter/pull/801) reflected a drop in coverage to the sessions controller,in CodeCov's comment. But after rebasing, now it shows a drop in the Annotations controller:

image

Why might CodeCov be varying in its reporting? cc @kaustubh-nair 

Thank you!

### PR DESCRIPTION
**Pull request by kaustubh-nair**
Fixes #810 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

[//]: # (To mark checkbox write 'x' within the square brackets)

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/mapknitter-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by kaustubh-nair** [comment]
I guess that worked?

**Comment by kaustubh-nair** [comment]
Let's try one last time

**Comment by kaustubh-nair** [comment]
Nice!
@jywarren I've think got it fixed
I ran the tests thrice with no decrease in coverage. Let's see how this goes

File: .travis.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ install:
   - bundle check --path vendor/.bundle/ || bundle install --path vendor/.bundle/
   - yarn check || yarn install
   - bundle exec rake db:setup || bundle exec rake db:migrate
+  - rm -rf test/reports
```
Comment (alaxalves) [inline .travis.yml]: There's no need to add this additional step to Travis, causing the build to be slower, I think you could just add test/reports to .gitignore (if its not there already), then remove this folder if its commited. But let's hear from @jywarren.

Comment (kaustubh-nair) [reply .travis.yml]: It's already in .gitignore 
@alaxalves Can you help removing it from git tracking?

**Comment by alaxalves** [comment]
I don't think this has worked :( check out codecov @kaustubh-nair

**Comment by kaustubh-nair** [comment]
Huhhhhhh

**Comment by kaustubh-nair** [comment]
Okay I guess this fixed it? I'm still not sure what the exact problem is. I don't know how this is possible but is travis caching any old reports maybe? @alaxalves?
Anyway @jywarren let's merge this and see how it goes
Thanks!

**Comment by alaxalves** [comment]
> Okay I guess this fixed it? I'm still not sure what the exact problem is. I don't know how this is possible but is travis caching any old reports maybe? @alaxalves?
> Anyway @jywarren let's merge this and see how it goes
> Thanks!

Travis doesn't cache old reports because basically we "tell" him what to cache and since we have deleted the reports from git tracking it won't cache. Also everytime Travis uses a different runner and caching is made through a generated .tar.gz pack. But this seems to be working now. Maybe we had to explicitly set the formatter to be codecov.

**Comment by jywarren** [comment]
Ok cool! Great work piecing this together. Did you want to merge this now? 🎉👍🏽

**Comment by kaustubh-nair** [comment]
@jywarren Yes lets merge this now

**Comment by jywarren** [comment]
👍

**Pull request merged**